.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "presets": [
-    ["es2015", { "modules": false }],
```

Comment:
I also tried this in my update dependencies PR, but didn't have any real effect. Did you try to recompile the `bundle.js` file and saw any differences? Maybe I was doing it wrong!

Reply:
Actually I was running into failures after upgrading to react 16, some JS tests where failing due to something about `es2015` not found. So I cherry-pick the commit from your branch and it fixed it... :thinking: